tracer/Datadog.Trace.proj

```diff
@@ -100,7 +100,7 @@
   <Target Name="RestoreSamplesForPackageVersionsOnly" Condition="'$(TestAllPackageVersions)'=='true'">
     <!-- Filter the sample projects by TargetFramework -->
     <ItemGroup>
-      <PackageVersionSampleCompile Include="@(PackageVersionSample)" Condition="'%(PackageVersionSample.TargetFramework)' == '' or '%(PackageVersionSample.TargetFramework)'=='$(TargetFramework)'" />
+      <PackageVersionSampleCompile Include="@(PackageVersionSample)" Condition="('$(IsAlpine)' != 'true' OR '%(PackageVersionSample.SkipAlpine)'!='true') AND ('%(PackageVersionSample.TargetFramework)' == '' or '%(PackageVersionSample.TargetFramework)'=='$(TargetFramework)')" />
```

Comment:
I'm convinced i'm missing something obvious cause I'm jetlagged today, so sorry in advance :) 
but I would have thought we should have something like
`!('$(IsAlpine)' == 'true' AND '%(PackageVersionSample.SkipAlpine)'=='true')`

Reply:
Those are functionally equivalent, as per [De Morgan's Law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) 😉 The difference is, mine checks that `IsAlpine` is anything _except_ `true` (so `False`, `false` etc work), and similarly for `SkipAlpine`.

Reply:
I really should have stayed in bed all day.